Directory.Build.targets

```diff
@@ -17,4 +17,9 @@
   </PropertyGroup>
   <Import Project="$(SourceRoot)src/Orleans.CodeGeneration.Build/build/Microsoft.Orleans.OrleansCodeGenerator.Build.targets" Condition=" '$(OrleansBuildTimeCodeGen)' == 'true' " />
   <!--End Orleans -->
+
+  <!-- Set InformationVersion here, since $(Version) is already set at this point. -->
+  <PropertyGroup>
+    <InformationalVersion>$(Version). Commit Hash: $(GitHeadSha)</InformationalVersion>
```

Comment:
why is this in targets and not props file?

Reply:
Ah, it's because this runs after the Version variable is set? I just read the comment above :)